src/marqo/tensor_search/tensor_search_logging.py

```diff
@@ -1,9 +1,14 @@
 import logging
-
+from marqo.tensor_search.utils import read_env_vars_and_defaults
 def get_logger(name):
-    logging.basicConfig() 
+    logging.basicConfig()
     logger = logging.getLogger(name)
-    logger.setLevel(logging.INFO)
+
+    log_level = read_env_vars_and_defaults("MARQO_LOG_LEVEL")
+    if log_level == "warning":
+        logger.setLevel(level=logging.WARNING)
+    elif log_level == "info":
+        logger.setLevel(level=logging.INFO)
```

Comment:
1. Make sure we mention there are only 2 log levels in the documentation.
2. Can you raise an error if an unsupported log level is provided? In the error mention what log levels are supported, and link to relevant docs.
3. Can you make the log levels case-insensitive?

Reply:
Thanks Li!